src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  &--4 {
+    display: flex;
+    gap: 2px;
+    align-items: center;
+    padding-left: $padding-left;
+    width: 96px;
+    height: 16px;
+  }
```

Comment:
You need to implement the modifier logic, use styles for stars from the previous task